x/axelarnet/keeper/msg_server.go

```diff
@@ -82,13 +82,14 @@ func (s msgServer) ConfirmDeposit(c context.Context, req *types.ConfirmDepositRe
 	ctx := sdk.UnwrapSDKContext(c)
 
 	depositAddr := nexus.CrossChainAddress{Address: req.DepositAddress.String(), Chain: exported.Axelarnet}
+	amount := s.bank.GetBalance(ctx, req.DepositAddress, req.Denom)
```

Comment:
I don't think the logic below fails if `amount` is `0`? Seems like we'll create a pending transfer with amount `0`.

Reply:
good point, gonna add a check